noxfile.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+
+from __future__ import absolute_import
+
+import nox
+import os
+# import shutil
+
+
+BLACK_VERSION = "black==19.3b0"
+BLACK_PATHS = ["docs", "google", "tests", "noxfile.py", "setup.py"]
```

Comment:
Looks like these are the paths from #466 still?

```suggestion
BLACK_PATHS = ["django_spanner", "examples", "spanner_dbapi", "tests", "noxfile.py", "setup.py"]
```

Reply:
Sorry, this should've been commented as well, since we are to implement `blacken` later.